lib/membrane/rtcp/parser.ex

```diff
@@ -58,6 +80,14 @@ defmodule Membrane.RTCP.Parser do
   @impl true
   def handle_event(:output, %RTCPEvent{} = event, _ctx, state) do
     buffer = %Buffer{payload: RTCP.Packet.serialize(event.rtcp)}
+
+    Membrane.TelemetryMetrics.execute(
+      @rtcp_sent_telemetry_event,
```

Comment:
This might be a bit problematic, as it includes FIRs, PLIs, NACKs, REMB (when/if implemented) etc in addition to receiver reports.
1. Do you thing such aggregated statistic is helpful?
2. I guess the number of SRs can be estimated by substraction `RTCP - FIR - NACK`, but maybe we should have a separate metric for that

Reply:
1. I think this comment answers it pretty well: https://github.com/membraneframework/membrane_rtp_plugin/pull/142#discussion_r1108397831